tests/input/concatenation_spacing.php

```diff
@@ -27,9 +27,9 @@
 
 $string = $doctrine . $rulez. 'even' .'more'.$string;
 
-$string.=$doctrine;
 $string .= $doctrine;
-$string .=$doctrine;
+$string .= $doctrine;
```

Comment:
I'd say we should change this but increase the number of expected violations.

Reply:
I thought that it doesn't matter anyway as this test is focused on an entirely different thing, but sure I will change it.